src/main/webapp/dev/js/common/questionConstSum.js

```diff
@@ -70,10 +70,28 @@ function removeConstSumOption(index, questionNum) {
     }
 }
 
+function toggleConstSumDistributePointsOptions(checkbox, questionNum) {
+    if (checkbox.checked) {
+        $(`#constSumDistributePointsSelect-${questionNum}`).prop('disabled', false);
+        $(`#constSumDistributePointsOptions-${questionNum}`).attr('value',
+                $(`#constSumDistributePointsSelect-${questionNum}`).prop('value'));
+    } else {
+        $(`#constSumDistributePointsSelect-${questionNum}`).prop('disabled', true);
+        $(`#constSumDistributePointsOptions-${questionNum}`).attr('value', 'None');
```

Comment:
for this part i think you can afford to use a few more lines to extract some variables and give them proper names to make the code more comprehensive. And on a random note can i ask y for some parts u use `prop(value)` to get the value, instead of `attr(value)`?

Reply:
1. Can I ask what you mean by "extract some variables and give them proper names"? Here, I am only referencing two items - `constSumDistributePointsSelect` and `constSumDistributePointsOptions`. I cannot immediately see how else I can further simplify this (unless you mean the naming)...

2. Admittedly the division between `prop`  and `attr` is somewhat nebulous. However, I did this mainly because I drew inspiration from the format for MCQ/MSQ questions. Besides, I have also read that:

>`attr()` gives you the value of element as it was defined in the html on page load. It is always recommended to use `prop()` to get values of elements which is modified via javascript/jquery , as it gives you the original value of an element's current state. 

so I thought that the MCQ/MSQ way of doing this was fine.